lib/holidays/use_case/context/dates_driver_builder.rb

```diff
@@ -15,6 +15,8 @@ def call(start_date, end_date)
           clean(dates_driver)
         end
 
+        #FIXME Why is the date_driver set to optional as the first param? That's
+        # just plain wrong...
         def build(dates_driver = {}, date)
```

Comment:
Oh, I was sure that *I* did this! I didn't realize it was added as part of the `next_holidays` work!

It's not a probem at the moment, just confusing. We can deal with it later. 👍